examples/decoding/plot_simulated_data.py

```diff
@@ -41,9 +41,9 @@ def create_simulation_data(snr=0, n_samples=2 * 100, size=12, random_state=1):
     w[-roi_size:, -roi_size:, 0:roi_size] = 0.5
```

Comment:
I think this file should be kept unchanged because this changes are not really related to this PR or are they?